services/storeNewBookUseCase.js

```diff
@@ -0,0 +1,52 @@
+const mapper = require('../middleware/booksMapper');
+const logger = require('../middleware/logger');
+const booksRepository = require('../repositories/booksRepository');
+const authorsRepository = require('../repositories/authorsRepository');
+const authorModel = require('../models/domain/authorModel');
+
+async function addNewAuthor(newAuthor){
+    logger.logRequest('addNewAuthor requested');
+    authorId = await authorsRepository.addNewAuthor(newAuthor.name, newAuthor.surname);
+
+    return authorId;
+}
+
+function addNewBook(newBook, authorId){
+    booksRepository.addNewBook(newBook, authorId);
+}
+
+async function isBookInStore(isbn){
+    let id = await booksRepository.findBookByIsbn(isbn);
+    logger.logData(id[0]);
+    
+    return id[0] === undefined;
+}
+
+async function isAuthorInStore(author){
+    let id = await authorsRepository.findAuthorByName(author);
+    
+    return id[0] === undefined;
```

Comment:
użyj tej meody z Utilsów, isNullOrUndefined

Reply:
poszukałem w dokumentacji Node.js, funkcja jest przestarzała (deprecated), wskazane jest używanie tak jak mam wyżej, mogę to wyeksportować do prywatnej metody authorExistsInStore